### PR TITLE
Styles home

### PR DESCRIPTION
**Pull request by mixcas**
- sytles featured post
- styles category names in the grid posts
- styles noticias and portadazza
- font sizes

### CODE REVIEW

File: front-page.php [hunk 1]
```diff
@@ -32,23 +32,15 @@
...
 
     <?php if ($subtitle) { ?>
-          <div class="featured-subtitle col s3">
-            <span class="rotate-text font-condensed"><?php echo $subtitle; ?></span>
-          </div>
+          <div id="featured-post-subtitle" class="rotate-text font-condensed js-fix-widows"><h4><?php echo $subtitle; ?></h4></div>
     <?php } ?>
 
-          <div class="col s12"><?php the_post_thumbnail(); ?></div>
+          <?php the_post_thumbnail('', array( 'id' => 'featured-post-image') ); ?>
```
Comment (varffs) [inline front-page.php:43]: what is this? why blank string then array here?

File: front-page.php [hunk 2]
```diff
@@ -175,9 +165,10 @@
 
       <div class="col s8">
 
-        <img src="https://placeholdit.imgix.net/~text?txtsize=50&txt=AD&w=400&h=400">
-
-        <img src="https://placeholdit.imgix.net/~text?txtsize=50&txt=AD&w=400&h=400">
+        <div class="feed-post-container col s6">
```
Comment (varffs) [inline front-page.php]: why is there a s6 inside an s8?

File: functions.php [hunk 3]
```diff
@@ -110,7 +111,7 @@ function get_instagram_feed($instagram_handle) {
       $feed = array_slice($feed, 0, 5);
 
       // Set response item's as transient with expiration time of 30 min
-      set_transient( 'instagram_feed_' . $instagram_handle, $feed, 30 * 'MINUTE_IN_SECONDS' );
+      set_transient( 'instagram_feed_' . $instagram_handle, $feed, 3 * 'MINUTE_IN_SECONDS' );
```
Comment (varffs) [inline functions.php:114]: is this on purpose?

File: front-page.php (hunk 2 above)
Comment (mixcas) [inline front-page.php]: i explained this already :/

**Comment by varffs** [comment]
missing `GothamHTF-MediumCondensed.woff2`

File: functions.php (hunk 3 above)
Comment (mixcas) [inline functions.php:114]: yes

Comment (mixcas) [inline functions.php:114]: wrong branch i guess

File: front-page.php (hunk 2 above)
Comment (varffs) [inline front-page.php]: the thing about advert widths and floating. why does the s6 need to be used tho? cant `.feed-post-container` just float?

Comment (mixcas) [inline front-page.php]: cuz `.s6` defines its width since its tied to the grid

Comment (varffs) [inline front-page.php]: oh wait. then better to have an empty s2 as the spacing element. then you don't need to override anything

Comment (mixcas) [inline front-page.php]: mmmm yeah, i think that should work

Comment (mixcas) [inline front-page.php]: yeah it must work and its probably better that way

Comment (varffs) [inline front-page.php]: `.col` should have `min-height: 1px` tho to force width

Comment (mixcas) [inline front-page.php]: i didnt know, whos fixing it? u or me?

**Comment by varffs** [comment]
the fonts are all def pretty screwed for whatever reason. I think that should just be another branch to fix it

**Pull request merged**